contracts/src/everlong/Everlong.sol

```diff
@@ -0,0 +1,71 @@
+//      ####################++++++++++++++++++++++++++++++++++#####
+//   #####+# #+############++++++++++++++++++++++++++++++++++++##############
+// ##+    ++##############+++++++++++++++++++++++++++++++++-++++###################
+// ######################++++++++++++++++++++++++++++++++++-++++##########  ####+ #
+// +#++++###############++++++++++#######++++++++++++++++++++++++#######      #####
+//     +########+######++++++++++++++++++++++++++++++++++++++++++####           ###
+//    ####  ###########++++++--+++++##++++++++++++++##+++++++++++####             #
+//  +#################+++++--+++++++###+#++++++++++++####++++++++#######
+// +##################++++++++++++++++++##+++++++++++###+++++++++#########
+// ##################++++++++++++++++++++++++++++++++++###+++++++###############
+// ##################++-+++++++++++++++++++++++++++++++##+++++++###################
+//   ###    #####++++++-++++++++++++++++++++++++++++++++++++++++##########  #######
+//  ###      ###+++++++++---+++++++++++++++-+++-++++++++++++++++######### #########
+//          ###++++++++------++++++++++++###++-++++++++--+++++++#############
+//        #####+++++++----------+++++++++++++++##++++-------+++##########
+//    #########++++++----------+++++++++++++++++++++++------+++#########
+// #############+++++----------+++++++++++++++++++++++++---+++###########
+// #############++-++----------+++++++++++++++++++++++-+---++###    ## ###
+// ### ###       ++++----------+++++++-++-++++++-+++++-----++###        ###
+//                 ++----------+++#+++---------+++--++-+---++#####       ####
+//                #+----------+++++++++++++--+--+++-------+-++#####       ####
+//              ####+---------++++++++++++++++++#+++-----++-++#######      ####
+//            ######+----------------++++++++++++++-----++--+    #####       ###
+//          ##### #+++------+++-------+++++++--+++-----++-###       ###      #####
+//         ###    ++--+-----+++++++++++++---------+---++-##++###    +##
+//        ##+   ###+--------+++++++++++++++++--+++---++++      ++#######
+//     ####+    ###+-----------+++++++++++++++++++++++++            #  #
+//           ++##++++---------------+++++++++++++++++
+//          ++## +++++------------+++++++++++++++++++
+//              +++++++-----------+++++++++++++++++++
+//          ++###++++++---------------++++++++++++++
+//       +++#####+++++++++--------------++++++++++++
+// #+ +++#######+++++++++++---------------+++++++++#+
+// ###########++++++++++++++-------------++++++++++###
+///
+/// @author DELV
+/// @title Everlong
+/// @notice A money market powered by Hyperdrive.
+/// @custom:disclaimer The language used in this code is for coding convenience
+///                    only, and is not intended to, and does not, have any
+///                    particular legal or regulatory significance.
+contract Everlong is EverlongAdmin, EverlongERC4626, EverlongPositions {
+    constructor(
+        address underlying_,
```

Comment:
Nit: Name is the first argument for most of our other contracts (should be all but the factory for historical reasons). Can we put name and symbol first?